### DISCUSSION TITLE
Centered weapons mod  (no arms) WIP

### DISCUSSION THREAD

**Discussion by SniperGoth** [redeclipse/base#1422] [Maps and Content Showcase]
This is a very basic HUD arm modification i made, in which you have no arms, and only the weapons displaying, as if it was an older FPS.

Sword, and claw don't work because they will require swing animations, which i'll do later, but all other weapons work without issues, so, until then, try to avoid using them.

![20230506_153115](https://user-images.githubusercontent.com/24359960/236642235-677cf325-a803-4864-98c2-f37dcfd2c7d6.png)
![20230506_154458](https://user-images.githubusercontent.com/24359960/236642241-23c1a353-06d9-4904-8379-2fb3b7fe2a30.png)
![20230506_155002](https://user-images.githubusercontent.com/24359960/236642246-6055428a-4819-4d24-81df-026dfdb7fe5c.png)
![20230506_160332](https://user-images.githubusercontent.com/24359960/236642389-c2fc59da-d766-4fb7-bd03-b5d1aa6edc63.png)
![20230506_155433](https://user-images.githubusercontent.com/24359960/236642390-8e70767e-6976-4c11-8ff5-8f8fb4324078.png)

Currently, it works on dev version, but maybe should work on vanilla 2.0 (i need to check it first).

Installation:
On windows, place the "old-school" zip file under my games/red eclipse  and it should be good to go.
As for linux and mac, i'm not sure.

[old-school.zip](https://github.com/redeclipse/base/files/11413170/old-school.zip)

**Comment by thevibingcat** [comment]
the location of where to put it on linux is
`/home/<username>/.local/share/Steam/steamapps/compatdata/967460/pfx/drive_c/users/steamuser/Documents/My Games/Red Eclipse`
